src/test/helpers.go

```diff
@@ -238,3 +247,28 @@ func SetupServer(ctx context.Context, t *testing.T) error {
 	}()
 	return nil
 }
+
+// NewTestDB creates a new in-memory SQLite database instance for testing purposes.
+func NewPostgresTestDB(t *testing.T) (sqlmock.Sqlmock, *db.PostgresDb) {
```

Comment:
this test is not used here, you could add some unit tests to test that the added functionality works as expected?